packages/twenty-front/src/modules/auth/sign-in-up/hooks/useSignInUpForm.ts

```diff
@@ -38,9 +38,15 @@ export const useSignInUpForm = () => {
   const isDeveloperDefaultSignInPrefilled = useRecoilValue(
     isDeveloperDefaultSignInPrefilledState,
   );
-  const [searchParams] = useSearchParams();
+  const [searchParams, setSearchParams] = useSearchParams();
   const prefilledEmail = searchParams.get('email');
 
+  if (isDefined(prefilledEmail)) {
```

Comment:
this does not seem to be very "react" compliant. This is a side effect so this should be a useEffect.

However, we want to avoid as much useEffects as we can.
Could you describe more the issue so I can assist in finding a solution here :)